pac4j-core/src/test/java/org/example/DummyValue.java

```diff
@@ -0,0 +1,37 @@
+package org.example;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * This class should be in a package different from org.pack4j.
+ * This is needed to test untrusted package deserialization feature
```

Comment:
Please add the `@since` and `@author` annotations.

Reply:
Just woke up from holidays...
Ok, will do that on the weekends.